awx/main/tests/functional/api/test_inventory.py

```diff
@@ -624,8 +624,10 @@ def test_patch_constructed_inventory(self, constructed_inventory, admin_user, pa
         assert inv_src.update_cache_timeout == 54
         assert inv_src.limit == 'foobar'
 
-    def test_patch_constructed_inventory_generated_source_limits_editable_fields(self, constructed_inventory, admin_user, project, patch):
-        inv_src = constructed_inventory.inventory_sources.first()
+    def test_patch_constructed_inventory_generated_source_limits_editable_fields(
+        self, constructed_inventory, admin_user, project, patch, compute_cloud_inventory_sources
```

Comment:
`compute_cloud_inventory_sources` would need to be an existing fixture for this to work.

Reply:
This has been resolved here: https://github.com/ansible/awx/pull/15537/commits/23b6ce18278e1e951ef38634fe48ffa9a8d87129

Reply:
Leaving this open since I'd like to try making a fixture that would be able to achieve this.

Reply:
FYI there's a number of test functions in the same module that still have this non-reverted.